lib/lotus/entity.rb

```diff
@@ -235,6 +252,46 @@ def update(attributes={})
       end
     end
 
+    # Getter for hash of changed attributes.
+    # Return empty hash, if there is no changes
+    #
+    # @since 0.3.0
+    #
+    # @example
+    #   @changed_attributes # => {:title => 'New title'}
```

Comment:
What do you think about amending the `@example` so that it include entity set/changes of attribute title? I believe it would help user to have a full picture of the workflow.